### DISCUSSION TITLE
I want to use Airsim exe files for eyetracking with HTC vive, anybody could do that to help me?

### DISCUSSION THREAD

**Discussion by Nilzfar** [microsoft/AirSim#3794] [Support Q&A]
# I want to use Airsim exe files for eyetracking with HTC vive, anybody could do that to help me?

**Comment by clovett** [comment]
Presumably you plan to use the [HTC Vivie Eye Tracking SDK](https://developer.vive.com/resources/vive-sense/sdk/vive-eye-and-facial-tracking-sdk/), with it's unreal plugin and use the eye tracking info to do something in AirSim, what do you want the eyetracking to do, look around, or control the drone?

**Comment by Nilzfar** [comment]
> Presumably you plan to use the [HTC Vivie Eye Tracking SDK](https://developer.vive.com/resources/vive-sense/sdk/vive-eye-and-facial-tracking-sdk/), with it's unreal plugin and use the eye tracking info to do something in AirSim, what do you want the eyetracking to do, look around, or control the drone?

well, yes I want to just extract where my subject is looking in real-time in a scene. I am using HTC vive pro , do you know if there is a plug-in for this in Airsim binaries?

**Comment by lovettchris** [comment]
I think you should first look for Unreal support for HTC Vive.  AirSim is just an unreal plugin, and I assume HTC Vive support is also an unreal plugin, so presumably you simply add the 2 plugins together and you get what you want.  So look for some unreal HTC Vive demo environments, then I can show you how to add AirSim to that.

**Comment by Nilzfar** [comment]
> I think you should first look for Unreal support for HTC Vive. AirSim is just an unreal plugin, and I assume HTC Vive support is also an unreal plugin, so presumably you simply add the 2 plugins together and you get what you want. So look for some unreal HTC Vive demo environments, then I can show you how to add AirSim to that.

sorry for my mistake, my headset is HTC Vive Pro. I could use VR mode with the Vive pro in Airsim.NH but I don't know how to get data of eye tracker in binaries.
